lib/formtools/plugins/plugins-helpers.js

```diff
@@ -29,10 +29,6 @@ var defaults = function (options, isEmbeddedDocument) {
     // set up fields to an empty object as default
     options.fields = options.fields || {};
```

Comment:
I think this could go too? We're not using it for anything else as far as I can see or search for... Please try removing it and running the code to make sure it doesn't break.